src/pointification.jl

```diff
@@ -27,3 +27,7 @@ pointify(d::Data) = pointify(domain(d))
 
 # fallback with iterator of geometries
 pointify(geoms) = mapreduce(pointify, vcat, geoms)
+
+# utils
+_asvector(itr) = collect(itr)
+_asvector(vector::AbstractVector) = vector
```

Comment:
no need to create this additional function, just use collect directly on vector as well.

Reply:
This utility function is to avoid unnecessary `collect`s, because the vertices of `PolyArea`, `Ring` and `Rope` are `AbstractVector`s.